.github/workflows/ci.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.10", "3.11", "3.12"]
-        netbox-version: ["v4.0.0", "master"]
+        netbox-version: ["v4.1.1", "master"]
```

Comment:
Could you please set the version to v4.1.0 for testing against the first release of NetBox 4.1?

Reply:
Yes, I will do it.